### ISSUE TITLE
Highlight roads lacking speed limits

### ISSUE THREAD

**Issue by bryceco** [bryceco/GoMap#301]
User requested a switch for this. Might be useful along with other common tags like lanes=*. Also see #221.

**Comment by wtimme** [comment]
Sounds similar to the quest that was implemented in StreetComplete (see [this list][1]). If you look at that one, there seem to have been some issues, e. g. in the United States.

Before we implement this, we might want to do a bit of research.

[1]: https://wiki.openstreetmap.org/wiki/StreetComplete/Quests

**Comment by bryceco** [comment]
Agreed that using it for a quest looks problematic. It should still be safe to have it as an opt-in overlay.

**Comment by tordans** [comment]
I really like the approach in completionist that allows me to highlight different "problems" on the map. I use it for speed limits, reference-keys, surface, and so on. IMO, it would be great if we continued the "what is required for a release" discussion in the other thread …
